src/dsp_tools/clients/legal_info_client_live.py

```diff
@@ -123,9 +118,9 @@ def enable_unknown_license(self) -> None:
             log_and_raise_request_exception(err)
         if response.ok:
             return
-        if response.status_code == HTTPStatus.UNAUTHORIZED:
+        if response.status_code == HTTPStatus.FORBIDDEN:
             raise BadCredentialsError(
-                "Only members of a project or system administrators can enable licenses. "
+                "Only SystemAdmins or ProjectAdmins can enable licenses. "
```

Comment:
ProjectMembers cannot do that, only ProjectAdmins. See https://api.dasch.swiss/api/docs/#/Admin%20API/putAdminProjectsShortcodeProjectshortcodeLegal-infoLicensesLicenseiriEnable